suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -658,6 +658,20 @@ export const getTestnetSymbols = () => getTestnets().map(n => n.symbol);
 export const isBlockbookBasedNetwork = (symbol: NetworkSymbol) =>
     networks[symbol]?.customBackends.some(backend => backend === 'blockbook');
 
+export const isDebugOnlyAccountType = (
+    symbol?: NetworkSymbol,
+    accountType?: NonNullable<AccountType | undefined>,
```

Comment:
```suggestion
    accountType?: AccountType,
```

maybe this is enough?

Reply:
Maybe, I was playing around with types because I keep getting an error that isDebugOnlyAccountType doesn't exist on the accountTypeInfo. But this doesn't seem to affect it.